nest-tax-backend/src/card-payment-reporting/card-payment-reporting.service.ts

```diff
@@ -244,7 +244,7 @@ export class CardPaymentReportingService {
     return head + body + footer
   }
 
-  async generateAndSendPaymentReport(email: string, from?: Date) {
+  async generateAndSendPaymentReport(email: string[], from?: Date) {
```

Comment:
Now that it is no longer a single email, maybe `recipients` would be more sufficient, or perhaps even `recipientEmails`, but it's quite verbose. Applies not only here, but throughout the whole reporting and admin endpoint. Other suggestions are open for discussion.